CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 _..._
 
+## 1.3.0 - 2016-04-05
+
+- Added `Command` and `Options` abstract classes
+- Deprecated `AbstractCommand` and `CommandInterface`
```

Comment:
We may want to have a separate document noting intended migration paths for deprecations like these?